projects/demo/src/modules/components/input-phone/examples/import/import.md

```diff
@@ -1,21 +1,13 @@
 ```ts
-import {ReactiveFormsModule} from '@angular/forms';
-import {TuiInputPhoneModule} from '@taiga-ui/legacy';
-
 // ...
 
 @Component({
   standalone: true,
   imports: [
     // ...
-    ReactiveFormsModule,
-    TuiInputPhoneModule,
+    TuiInputPhone,
```

Comment:
Forgot where from and to update the HTML on setup tab – it still shows `tui-input-phone`